src/services/featurehelper.js

```diff
@@ -88,6 +89,13 @@ ngeo.FeatureHelper = function($injector, $filter) {
    */
   this.projection_;
 
+  /**
+   * Download service.
+   * @type {ngeo.Download}
+   * @private
+   */
+  this.download_ = $injector.get('ngeoDownload');
```

Comment:
For me the injector should be used when the dependency is optional (and in the gmf controller for simplification).
If it's not like that we can clarify it later...